### ISSUE TITLE
Can I get a screenshot of your system?

### ISSUE THREAD

**Issue by ssleert** [metafates/Workflow#1]

**Comment by metafates** [comment]
Yeah! My primary OS is macOS, which looks boring, trust me. *macOS is awful at customization.*
But... I run dual-boot [Asahi Linux](https://asahilinux.org/) with [Hyprland WM](https://hyprland.org/) on my m1 macbook, so here's some screenshots

![photo_2023-02-19_12-24-23](https://user-images.githubusercontent.com/62389790/219939777-9cb90751-2542-4390-b687-e746f18c031d.jpg)
![photo_2023-02-19_12-24-25](https://user-images.githubusercontent.com/62389790/219939780-83847dae-780f-4d44-a7da-6c2fdd6321b2.jpg)
![photo_2023-02-19_12-24-29](https://user-images.githubusercontent.com/62389790/219939781-21415c90-cc57-4caa-aaec-50a85d6258de.jpg)
![photo_2023-02-19_12-24-31](https://user-images.githubusercontent.com/62389790/219939782-4de23ea8-2a06-467f-97c6-03056044aa45.jpg)
![photo_2023-02-19_12-24-32](https://user-images.githubusercontent.com/62389790/219939783-f92b1fda-deea-4885-ae96-86eb4ecba26e.jpg)
![photo_2023-02-19_12-24-33](https://user-images.githubusercontent.com/62389790/219939784-27b2f454-c7cd-4f0d-b2fc-7b8ec9b36f70.jpg)

**Comment by ssleert** [comment]
Wow, that looks cool.
I wish I had been able to see you last week.
But I'm glad someone is using my useless fetch)

**Comment by metafates** [comment]
> But I'm glad someone is using my useless fetch)

Nitch is the best fetch tool in my opinion, I really like the way it looks 😛

**Comment by ssleert** [comment]
> > But I'm glad someone is using my useless fetch)
> 
> Nitch is the best fetch tool in my opinion, I really like the way it looks :stuck_out_tongue:

There is an idea to rewrite it to golang with a normal configuration and support for more systems 🙂

**Comment by metafates** [comment]
I think that it's being written in **Nim 👑** adds some flavor to it! Actually, this is the only Nim application I used

**Comment by ssleert** [comment]
Sorry for such a late message.

You said you are using an m1 macbook with **Asahi linux**. I started writing my own top a couple of weeks ago and I was interested in supporting m1/m2 processors.

Can I get the `/proc/cpuinfo` file from your Asahi installation?

**Comment by metafates** [comment]
Here you go

<details>
<pre><code>processor	: 0
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x022
CPU revision	: 1

processor	: 1
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x022
CPU revision	: 1

processor	: 2
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x022
CPU revision	: 1

processor	: 3
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x022
CPU revision	: 1

processor	: 4
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x023
CPU revision	: 1

processor	: 5
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x023
CPU revision	: 1

processor	: 6
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x023
CPU revision	: 1

processor	: 7
BogoMIPS	: 48.00
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 asimdfhm dit uscat ilrcpc flagm ssbs sb paca pacg dcpodp flagm2 frint
CPU implementer	: 0x61
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0x023
CPU revision	: 1
</code></pre>
</details>
